corehq/apps/aggregate_ucrs/views.py

```diff
@@ -11,7 +11,7 @@
 from corehq.apps.aggregate_ucrs.models import AggregateTableDefinition
 from corehq.apps.aggregate_ucrs.tasks import populate_aggregate_table_data_task
 from corehq.apps.domain.decorators import login_and_domain_required, login_or_basic
-from corehq.apps.userreports.sql import IndicatorSqlAdapter
```

Comment:
@czue can comment on this, but I suspect this was done because aggregate UCRs were never supposed to handle non-SQL adapters (assuming this was done when ES adapters were still a thing).

Don't think this change should be blocking. just noting in case aggregate UCRs and non-sql adapters ever become things we can use

Reply:
Thanks for the flag. Memory is a bit fuzzy, but I think I probably did explicitly want to enforce SQL and also agree this seems fine for now.